build/Dockerfile.ci

```diff
@@ -122,6 +128,9 @@ COPY --from=build /build/windows-machine-config-operator/ovn-kubernetes/go-contr
 # Copy windows_exporter.exe
 COPY --from=build /build/windows-machine-config-operator/windows_exporter/windows_exporter.exe .
 
+# Copy azure-cloud-node-manager.exe
+COPY --from=build /build/windows-machine-config-operator/cloud-provider-azure/azure-cloud-node-manager.exe .
+
```

Comment:
Why is the WMCB submodule being updated in this commit?

Reply:
My bad, I accidentally added it, and it caused the failure. Now the module is removed.